.travis.yml

```diff
@@ -8,7 +8,7 @@ matrix:
 
 script:
   - ./config/travis/run-checks.sh
-  - cd ./frontend; npm run lint src/**/*js; cd ..
+  - cd ./frontend; npm run lint src/**/*js; cd ..;
```

Comment:
Why this change?

Reply:
oh yeah, that was residue from the travis change, let me go fixit